hw/ip/alert_handler/alert_handler.core

```diff
@@ -8,37 +8,8 @@ description: "Alert Handler"
 filesets:
   files_rtl:
     depend:
-      - lowrisc:ip:tlul
-      - lowrisc:prim:all
-    files:
-      - rtl/alert_handler_reg_pkg.sv
-      - rtl/alert_handler_reg_top.sv
```

Comment:
moved to `alert_handler_reg.core`

Reply:
why move reg out of alert_hander. looks like for the other IP, reg is still in the same core file with RTL design.

Reply:
The rationale is listed in bug #1565. THis is to support top_earlgrey which has different parameterization than the "default" used for alert_handler in the IP area (and hence different reg top and pkg sources).